src/main/java/events/Reminder.java

```diff
@@ -1,228 +0,0 @@
-            }
-        }
-        catch (DateTimeParseException ex) {
-            event.getChannel().sendMessage(ex.getMessage()).queue();
-        }
-    }
-
-    public void printReminders(@Nonnull GuildMessageReceivedEvent event) {
-        String authorId = event.getAuthor().getId();
-        int numberOfReminder = 0;
-
-        for (int i = 0; i < reminders.size(); i++) {
-            if (reminders.get(i).get(1).equals(authorId)) {
-                numberOfReminder++;
-                if (numberOfReminder == 1) {
-                    event.getChannel().sendMessage("<@" + authorId + "> your reminders:").queue();
-                }
-
-                event.getChannel().sendMessage("ID: " + reminders.get(i).get(0) + ", date: " + reminders.get(i).get(3) + ", content: \"" + reminders.get(i).get(2) + "\"").queue();
-            }
-        }
-
-        if (numberOfReminder == 0)
-            event.getChannel().sendMessage("You have no reminders.").queue();
-        else
-            event.getChannel().sendMessage("That`s all.").queue();
-
-    }
-
-    private void deleteReminder(@Nonnull GuildMessageReceivedEvent event, String id) {
-        String authorID = event.getAuthor().getId();
-
-        for(int i = 0; i < reminders.size(); i++) {
-            if (reminders.get(i).get(0).equals(id) && reminders.get(i).get(1).equals(authorID)) {
-                event.getChannel().sendMessage("<@" + authorID + ">Reminder no. " + id + " was deleted (content: \"" +
-                        reminders.get(i).get(2) + "\", date: \"" + reminders.get(i).get(3) + ").").queue();
-                reminders.remove(i);
-                removeFromFile(event, id);
-            }
-        }
-
-        event.getChannel().sendMessage("There`s no reminder no. " + id +  ".").queue();
-    }
-
-    public static void deleteReminder(String id) {
-        for (int i = 0; i < reminders.size(); i++) {
-            if (reminders.get(i).get(0).equals(id)) {
-                reminders.remove(i);
-                removeFromFile(id);
-            }
-        }
-    }
-
-    private void saveToFile(GuildMessageReceivedEvent event, String[] splittedMessage) {
-        try (FileWriter output = new FileWriter(container, true)) {
-            String outputReminder = messageId + ";" + event.getAuthor().getId() + ";" + splittedMessage[1] + ";" + splittedMessage[2] + "\n";
-            output.append(outputReminder);
-        }
-        catch (IOException ex) {
-            ex.printStackTrace();
-        }
-    }
-
-    private void removeFromFile(GuildMessageReceivedEvent event, String id) {
-        String authorID = event.getAuthor().getId();
-
-        try (BufferedReader input = new BufferedReader(new FileReader(container))){
-            String actualLine;
-            String[] splittedMessage;
-            StringBuffer buff = new StringBuffer();
-
-            while((actualLine = input.readLine()) != null) {
```

Comment:
I tak czytasz wszystko, możesz to zastąpić Files::readAllLines, bo pliki nie są duże
https://docs.oracle.com/javase/8/docs/api/java/nio/file/Files.html#readAllLines-java.nio.file.Path-